app/javascript/gobierto_data/webapp/components/sets/Summary.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <Info />
+    <div class="gobierto-data-container-btn-download-data">
```

Comment:
Convert this button into a component (DownloadButton) if you're gonna use several times.